app/policies/page_policy.rb

```diff
@@ -0,0 +1,25 @@
+class PagePolicy < ApplicationPolicy
+  attr_reader :user, :page
+
+  def initialize(user, page)
+    @user = user
+    @page = page
+  end
+
+  def index?
+    @user.isAdmin?
```

Comment:
why `isAdmin?` and not `is_admin`?
(or even better `admin?`)

Reply:
@lucascaton , isAdmin? is used in the existing codebase, but this will be refactored later in the sprint :)